src/shared/varlink-io.systemd.Machine.c

```diff
@@ -131,4 +164,8 @@ SD_VARLINK_DEFINE_INTERFACE(
                 SD_VARLINK_SYMBOL_COMMENT("Machine uses a complex UID/GID mapping, cannot determine shift"),
                 &vl_error_NoUIDShift,
                 SD_VARLINK_SYMBOL_COMMENT("Requested information is not available"),
-                &vl_error_NotAvailable);
+                &vl_error_NotAvailable,
+                SD_VARLINK_SYMBOL_COMMENT("Requested operation is not supported"),
+                &vl_error_NotSupported,
+                SD_VARLINK_SYMBOL_COMMENT("There is no system bus in container"),
+                &vl_error_NoSDBus);
```

Comment:
hmm, we will sooner or later probably switch from D-Bus to varlink for this too, or try both, hence the error should be generic, and not encode D-Bus in its name.

maybe just "NoIPC"?